platform/tyan-s1706/platform.dts

```diff
@@ -0,0 +1,172 @@
+// SPDX-License-Identifier: GPL-2.0+
+/dts-v1/;
+
+#include "aspeed-g5.dtsi"
+#include "config.h"
+#include <dt-bindings/gpio/aspeed-gpio.h>
+
+/ {
+	model = "Tyan S1706";
+	compatible = "ocp,leopard-bmc", "aspeed,ast2500";
```

Comment:
This dts is a copy of the quanta one. I'll need to go over it better before getting this PR merged.

Reply:
Yeah, something like "tyan,s1706" should be fine there. You can get some inspiration on the DTS from these two:

- https://github.com/openbmc/linux/blob/dev-4.19/arch/arm/boot/dts/aspeed-bmc-facebook-tiogapass.dts
 - https://github.com/openbmc/linux/blob/dev-4.19/arch/arm/boot/dts/aspeed-bmc-intel-s2600wf.dts